api/app/controllers/api/v1/ingests_controller.rb

```diff
@@ -0,0 +1,53 @@
+module API
+  module V1
+    # Trigger ingestion processing via the API
+    class IngestsController < ApplicationController
+      before_action :set_project, only: [:create]
+      before_action :set_text, only: [:create]
+
+      resourceful! Ingestion do
+        @project&.ingestions
+        @text&.ingestions
+      end
+
+      # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+      def create
+        parent = @project || @text
+        @ingestion = ::Updaters::Ingestion
+          .new(ingestion_params)
+          .update_without_save(parent.ingestions.new)
+
+        @ingestion.creator = current_user
```

Comment:
Using `current_user` here and below is totally fine and fits with how things are done elsewhere in Manifold. This comes in on the API and the user who triggered the ingestion is the one who should be its creator / processor.